include/sdf/Link.hh

```diff
@@ -321,12 +321,24 @@ namespace sdf
     /// \sa bool Model::EnableWind
     public: bool EnableWind() const;
 
+    /// \brief Check if this link should be subject to gravity.
+    /// If true, this link should be affected by gravity.
+    /// \return true if the model should be subject to gravity, false otherwise.
+    /// \sa bool Model::EnableGravity
```

Comment:
```suggestion
```
`Model` doesn't have this API as it is not part of the `model.sdf` spec; this line can be removed